src/policy/keystone.rs

```diff
@@ -3,24 +3,176 @@
 //! This policy module enforces the Keystone policies, i.e. it enables the creation of user-level
 //! enclaves by leveraging PMP for memory isolation.
 
+#![allow(unused)]
```

Comment:
In general we don't want to add an `#![allow(unused)]` to the whole module, because then we can't see which functions or variables are not actually used.

Here I guess this is because of the FIDs as we probably wont use all of them? What I suggest for those situation is to create a new module inside this one and put the FID there, and limiting the `#![allow(unused)]` to that module.

For instance:

```rs

mod sbi {
#![allow(unused)]

/// Keystone EID & FIDs
///
/// See https://github.com/keystone-enclave/keystone/blob/80ffb2f9d4e774965589ee7c67609b0af051dc8b/sdk/include/shared/sm_call.h#L5C1-L6C1
const KEYSTONE_EID: usize = 0x08424b45;

// 1999-2999 are called by host
const CREATE_ENCLAVE_FID: usize = 2001;
const DESTROY_ENCLAVE_FID: usize = 2002;
const RUN_ENCLAVE_FID: usize = 2003;
const RESUME_ENCLAVE_FID: usize = 2005;

// 2999-3999 are called by enclave
const RANDOM_FID: usize = 3001;
const ATTEST_ENCLAVE_FID: usize = 3002;
const GET_SEALING_KEY_FID: usize = 3003;
const STOP_ENCLAVE_FID: usize = 3004;
const EXIT_ENCLAVE_FID: usize = 3006;
}
```

If there is more things which require `#![allow(unused)]` you can add individual `#[allow(unused)]` to those structs or functions. But don't forget to remove them once they become used :)

Reply:
I see, that make sense, I did the changes and added a todo so that I don't forget to remove the `#[allow(unused)]`